rewrite-test/src/main/kotlin/org/openrewrite/java/OrderImportsTest.kt

```diff
@@ -786,5 +787,128 @@ interface OrderImportsTest : JavaRecipeTest {
             }
         """
     )
+
+    @Disabled("Requires JavaProvenance on parsed test source files.")

```

Comment:
Placeholder for input.
The test requires a modification to the test harness to add JavaProvenance info to the source files.
Is it worth adding JavaProvenance to the test harness?